nanostack-event-loop/platform/arm_hal_timer.h

```diff
@@ -48,6 +48,31 @@ extern void platform_timer_disable(void);
  * \return 50us time slot remaining
  */
 extern uint16_t platform_timer_get_remaining_slots(void);
+/**
+ * \brief This function is API for enabling low resolution tick timer
+ *
+ */
+extern void platform_tick_timer_enable(void);
+/**
+ * \brief This function is API for registering low resolution tick timer callback
+ *
+ * \return -1 for false, success otherwise
+ */
+extern int8_t platform_tick_timer_register(void (*tick_timer_cb_handler)(void));
+/**
+ * \brief This function is API for starting the low resolution tick timer
+ *
+ * \param milliseconds define how many milliseconds time period will be started
```

Comment:
"-1 for failure". Be explicit that this gets called periodically until stop is called.